gradle/dependencies.gradle

```diff
@@ -130,6 +131,7 @@ libs += [
   apachedsJdbmPartition: "org.apache.directory.server:apacheds-jdbm-partition:$versions.apacheds",
   argparse4j: "net.sourceforge.argparse4j:argparse4j:$versions.argparse4j",
   bcpkix: "org.bouncycastle:bcpkix-jdk15on:$versions.bcpkix",
+  bcprov: "org.bouncycastle:bcprov-jdk15on:$versions.bcprov",
```

Comment:
Is the any advantage of this library compare to https://docs.oracle.com/javase/8/docs/api/java/util/Base64.Encoder.html

\cc @ijuma

Reply:
Actually, just realized that this library existed. :P 
Didn't know until later. Will remove this dependency.

Reply:
Sounds good.